### DISCUSSION TITLE
多个商品填同一个类目时报错

### DISCUSSION THREAD

**Discussion by zhanshen360** [Baiyuetribe/kamiFaka#50] [Bug反馈]
新增商品跟早些时候的商品属于同一个商品类目，我在新增商品时，把新增的商品选了以前商品的类目，可发型添加不了，报错！但你发现你搭建的是可以在同一个商品类目有多个商品的，我做错了什么吗?还是软件BUG？我用的1.7版！

**Comment by zhanshen360** [comment]
> 新增商品跟早些时候的商品属于同一个商品类目，我在新增商品时，把新增的商品选了以前商品的类目，可添加不了，报错！但我发现你搭建的是可以在同一个商品类目有多个商品的，我做错了什么吗?还是软件BUG？我用的1.7版！

**Reply by Baiyuetribe** [reply]
都1.7了还会不正常？我测试正常，注意新增商品所有参数都是必填项。
